crates/nu-command/src/conversions/into/int.rs

```diff
@@ -72,7 +73,12 @@ impl Command for SubCommand {
             ])
             .allow_variants_without_examples(true)
             .named("radix", SyntaxShape::Number, "radix of integer", Some('r'))
-            .switch("little-endian", "use little-endian byte decoding", None)
```

Comment:
Removing `--little-endian` makes this a bit harder to read (compared to `--endian little`